config/example.env

```diff
@@ -1,4 +0,0 @@
-REDIS_HOST=redis
-REDIS_PORT=6379
-REDIS_PASSWORD=123456
-SERVER_PORT=3000
```

Comment:
Why remove the example config? We may need it for dev setup instructions in the future.

Reply:
Because we have two example files one in the project root and this

Reply:
Because we have two example files one in the project root and this

Reply:
Because we have two example files one in the project root and this

Reply:
Because we have two example files one in the project root and this

Reply:
Because we have two example files one in the project root and this

Reply:
Because we have two example files one in the project root and this

Reply:
Because we have two example files one in the project root and this

Reply:
Because we have two example files one in the project root and this